common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/challengehandlers/PKeyAuthChallengeHandler.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.ui.embeddedwebview.challengehandlers;
+
+import android.os.Handler;
+
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationResponse;
+
+final class PKeyAuthChallengeHandler {
+    private Handler mRefHandler; //NOPMD
+
+    private AuthorizationResponse mResponse; //NOPMD
+
+    PKeyAuthChallengeHandler(Handler ref, AuthorizationResponse response) {
+        mRefHandler = ref;
+        mResponse = response;
+    }
+
+    void handleChallenge() {

```

Comment:
In general, I dislike the vagueness of method names like
`processFoo()`
`doBar()`
`handleBaz()`

because _every_ method processes, does, or handles something.

Can we get more specific?